lib/usdUfe/ufe/Global.h

```diff
@@ -44,10 +44,11 @@ struct USDUFE_PUBLIC DCCFunctions
     TimeAccessorFn      timeAccessorFn = nullptr;
 
     // Optional: default values will be used if no function is supplied.
-    IsAttributeLockedFn  isAttributeLockedFn = nullptr;
-    SaveStageLoadRulesFn saveStageLoadRulesFn = nullptr;
-    IsRootChildFn        isRootChildFn = nullptr;
-    UniqueChildNameFn    uniqueChildNameFn = nullptr;
+    IsAttributeLockedFn        isAttributeLockedFn = nullptr;
+    SaveStageLoadRulesFn       saveStageLoadRulesFn = nullptr;
+    IsRootChildFn              isRootChildFn = nullptr;
+    UniqueChildNameFn          uniqueChildNameFn = nullptr;
+    DefaultMaterialScopeNameFn defaultMaterialScopeNameFn = nullptr;
```

Comment:
New DCC optional function to provide the default material scope name. It none is set an appropriate default ("mtl") is used.